README.md

```diff
@@ -0,0 +1,85 @@
+## Budget App
+The Ruby on Rails capstone project (remember what they are?) is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+This id a Ruby on Rails application that allows the users to:
+
+- register and log in, so that the data is private to them.
+- introduce new transactions associated with a category.
+- see the money spent on each category.
+
+## Live Demo
+checkout the app [here](https://thawing-earth-92046.herokuapp.com) 
+
+## Video Presentation
+Watch the video presentation [here](https://www.loom.com/share/2bb894aff8194acc98da5b6e46f9d691)
+
+## Built With
+
+- Technologies used:
+  > Ruby
+  > Rails
+  > Tailwindcss
+  > postgresql
+
+
+## Getting Started
+
+### Prerequisites
+
+You need to have a browser and this tools installed:
+
+- Git
+- Ruby
+- MVC patterns
+
+### Setup
+
+To get a local copy up and running follow these simple example steps:
+
+```
+$ git clone git@github.com:tylher/c.git
+```
+
+```
+$ cd BudgetApp
+```
+
+```
+$ bundle install
+```
+
+### Usage
+
+To run the server:
+
+```
+$ rails s
+```
+To watch css changes
+
+```
+$ rails tailwindcss:watch
+```
+OR
+```
+$ ./bin/dev
+```
+To run tests,
+```
+$ rspec
+```
+
+## Authors
+
+👤 <a href="https://github.com/tylher" target="blank"><img align="center"
+      src="https://user-images.githubusercontent.com/57408419/163676887-390d6032-6720-42bb-ad16-f8e199d6f2fa.jpg"
+      alt="Taiwo Adediran" height="50" width="50"/></a> **Taiwo Damola Adediran**
+
+- GitHub: [@tylher](https://github.com/tylher)
+- Twitter: [@tylher123](https://twitter.com/tylher123)
+- LinkedIn: [Taiwo Adediran](https://www.linkedin.com/in/taiwo-adediran)
+  
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
```

Comment:
Please make sure that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).